package.json

```diff
@@ -9,8 +9,9 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@helpscout/javascript-sdk": "^0.0.11",
-    "@helpscout/ui-kit": "^0.0.37",
+    "@helpscout/javascript-sdk": "^0.1.0",
+    "@helpscout/ui-kit": "^0.1.0",
+    "@vitejs/plugin-basic-ssl": "^1.0.1",
```

Comment:
Should this be a `devDependency`?

Reply:
ah yes, thanks!

Reply:
fixed in 56c92b92ca79977d44e51176cf54a77635bdc75f